Meadowlark-js/package.json

```
@@ -59,6 +59,8 @@
     "test:e2e:debug": "cross-env DEBUG=testcontainers* npm run test:e2e:jest",
     "test:e2e:build": "npm run docker:build && npm run test:e2e:jest",
     "test:e2e:build:wsl1": "npm run docker:build:wsl1 && npm run test:e2e:jest",
+    "test:e2e:dev:setup": "cross-env TESTCONTAINERS_RYUK_DISABLED=true ts-node tests/e2e/setup/DevModeConfig.ts",
+    "test:e2e:dev:exit": "docker stop mongo-test opensearch-test meadowlark-api-test && docker rm -f mongo-test opensearch-test meadowlark-api-test && docker network prune -f",
     "start:local": "lerna run start:local",
```

Can we get something like this... ? And if I have the wrong command sequence, then that is all the more reason to help us with a one line version 😉 .

```suggestion
    "test:e2e:dev": "npm run test:e2e:dev:setup && npm run test:e2e:build && npm run test:e2e:exit",
    "start:local": "lerna run start:local",
```

---

I understand, but I don't think this adds value given that this is doing the same as running the e2e tests without setting the DEVELOPMENT_MODE, since it's setting the containers, running the tests and then cleaning it. 
In that case, one could run the tests with the setting "turned off".

Do you have an scenario in mind?